src/panels/lovelace/common/create-row-element.ts

```diff
@@ -27,6 +27,7 @@ import "../special-rows/hui-divider-row";
 import "../special-rows/hui-section-row";
 import "../special-rows/hui-weblink-row";
 import "../special-rows/hui-cast-row";
+import "../special-rows/hui-select-row";
```

Comment:
Let's not make this part of Lovelace, keep the components standalone. No need to use createElement.